internal/validation/suite.go

```
@@ -61,15 +58,14 @@ func RunInstanceLifecycleValidation(t *testing.T, config ProviderConfig) {
 		t.Skip("Skipping validation tests in short mode")
 	}
 
-	apiKey := os.Getenv(config.EnvVarName)
-	if apiKey == "" {
-		t.Skipf("%s not set, skipping %s validation tests", config.EnvVarName, config.ProviderName)
-	}
-
-	client := config.ClientFactory(apiKey)
 	ctx, cancel := context.WithTimeout(context.Background(), 15*time.Minute)
 	defer cancel()
 
+	client, err := config.Credential.MakeClient(ctx, "")
+	if err != nil {
+		t.Skipf("Failed to create client for %s: %v", config.ProviderName, err)
```

we should fail here instead of skip